<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="no_parallel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>307</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Без распараллеливания</string>
    </property>
   </widget>
   <widget class="QPushButton" name="time_mes_btn_r">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>289</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Скорость алгоритмов (с распал.)</string>
    </property>
   </widget>
   <widget class="QLabel" name="threads_text_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>142</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Размер файла (в кб)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="parallel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>307</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>С распараллеливанием</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exec">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>291</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Обработать текст</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>100</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="threads_entry">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>180</number>
    </property>
   </widget>
   <widget class="QLabel" name="threads_text">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>142</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Кол-во потоков</string>
    </property>
   </widget>
   <widget class="QPushButton" name="time_mes_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>289</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Скорость алгоритмов (без распал.)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>387</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
